ee/api/test/test_organization.py

```diff
@@ -0,0 +1,55 @@
+from unittest.mock import patch
+
+from django.test.utils import tag
+
+from posthog.models.organization import Organization, OrganizationInvite, OrganizationMembership
+from posthog.models.team import Team
+
+from .base import APILicensedTest
+
+
+class TestOrganizationEnterpriseAPI(APILicensedTest):
+    def test_create_organization(self):
+        response = self.client.post("/api/organizations/", {"name": "Test"})
```

Comment:
Any signed in user. Now, this is actually pretty clear on self-hosted, where the license is per-instance and we can just block creating additional organizations without a license, but unsure what to do on Cloud, where billing is per-organization. So we can't really make multiple organizations a paid feature. And also can't make multiple projects then, because you can create an organization as a surrogate of that anyway. Sentry for instance, which is quite similar in how its built, puts no restrictions on org/proj creation. Thoughts?

Reply:
Well regarding on whether the feature should be restricted, I think at the very least we should start enabling it from the Starter plan, might even make sense to push to Hobby. @timgl thoughts?

As for creating orgs, what you described makes sense.